### ISSUE TITLE
CPAN distributions should contain files META.yml and META.json

### ISSUE THREAD

**Issue by pawal** [dotse/zonemaster#431]
From a CPAN user:

CPAN distributions should contain files META.yml and META.json. They are generated by your build tool when creating distribution.

Build tool for IIS/Zonemaster-v1.0.7.tar.gz was detected as Module::Install. Please upgrade it and ExtUtils::MakeMaker to latest version.

You can create your distribution with these files using `make dist` (dmake on Windows).

**Comment by pawal** [comment]
https://rt.cpan.org/Ticket/Display.html?id=108252

    Remove inc/ directory and let Module::Install recreate it. If I understand correctly, META.* is created only if .author directory is present in inc/.

**Comment by mtoma** [comment]
- Removed inc/
- Removed one of the files created Module::Install package from MANIFEST
- Updated Module::Install and ExtUtils::MakeMaker to the latest version
- Installed CPAN::Meta (http://search.cpan.org/~bingos/ExtUtils-MakeMaker-7.10/lib/ExtUtils/MakeMaker.pm#Module_Meta-Data_(META_and_MYMETA))

Then run: rm -rf inc/ && perl Makefile.PL && make dist
The tar.gz file contains META.yml

Not sure about the reason why the inc/ directory is in the repository. It is created by the "perl Makefile.PL". It removed it in the commit. Tell me if you are aware of some reason to keep it in the repo.

**Comment by pawal** [comment]
Regarding your second point: which file?
Third point, you mean on your system, right?
 I will try to replicate your steps. Thanks!

**Comment by mtoma** [comment]
This "Found extra file inc/Module/Install/Scripts.pm" was blocking the make dist process at fitrs. So I removed the inc/Module/Install/Scripts.pm file from the MANIFEST

I updated the Module::Install and ExtUtils::MakeMaker modules as suggested bu the CPAN RT ticket on my system. Yes

But as discussed on the pull request issue we should avoid to use static MANIFEST file as this is a task that can be done when releasing the package to CPAN.
